lib/faker/name/it.ex

```diff
@@ -0,0 +1,279 @@
+defmodule Faker.Name.It do
+  import Faker, only: [sampler: 2]
+
+  @moduledoc """
+  Functions for name data in Italian
+  """
+
+  @doc """
+  Returns a complete name (may include a suffix/prefix or both)
+
+  ## Examples
+
+      iex> Faker.Name.It.name()
+      "Sig.ra Agave Bianchi"
+      iex> Faker.Name.It.name()
+      "Gennaro Mazza"
+  """
+  @spec name() :: String.t()
+  def name, do: name(Faker.random_between(0, 9))
+  defp name(0), do: "#{prefix()} #{first_name()} #{last_name()}"
+  defp name(_), do: "#{first_name()} #{last_name()}"
+
+  @doc """
+  Returns a random first name
+
+  ## Examples
+
+      iex> Faker.Name.It.first_name()
+      "Azalea"
+      iex> Faker.Name.It.first_name()
+      "Dionigi"
+      iex> Faker.Name.It.first_name()
+      "Agave"
+  """
+  @spec first_name() :: String.t()
+  sampler(:first_name, [
+        "Abramo", "Acantha", "Ace", "Achille", "Ada", "Adamo",
```

Comment:
You should run the formatter for this file to keep consistency:`mix format lib/faker/name/it.ex`

Reply:
I was sure I had the formatter running on save like in other projects but it didn't run. Fixed it.
I updated the Changelog. Please check that my update is in the right place.